docs/en/setup/backend/backend-bookkeeper-monitoring.md

```diff
@@ -0,0 +1,61 @@
+# BookKeeper monitoring
+
+SkyWalking leverages Prometheus to collect metrics data from the BookKeeper and leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+Kafka entity as a `Service` in OAP and on the `Layer: BOOKKEEPER.
+
+## Data flow
+
+1. BookKeeper exposes metrics in Prometheus format.
+2. OpenTelemetry Collector fetches metrics from BookKeeper cluster via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via OpenTelemetry gRPC exporter.
+3. The SkyWalking OAP Server parses the expression with [MAL](../../concepts-and-designs/mal.md) to
+   filter/calculate/aggregate and store the results.`
+
+## Setup
+
+1. Set up [BookKeeper Cluster](https://bookkeeper.apache.org/docs/deployment/manual). 
+2. Set up [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/getting-started/#kubernetes). The example
+   for OpenTelemetry Collector configuration, refer
+   to [here](../../../../test/e2e-v2/cases/pulsar/otel-collector-config.yaml).
+3. Config SkyWalking [OpenTelemetry receiver](opentelemetry-receiver.md).
+
+## BookKeeper Monitoring
+
+Bookkeeper monitoring provides multidimensional metrics monitoring of BookKeeper cluster as `Layer: BOOKKEEPER` `Service` in
+the OAP. In each cluster, the nodes are represented as `Instance`.
+
+### BookKeeper Cluster Supported Metrics
+
+| Monitoring Panel             | Metric Name                                                      | Description                                                | Data Source         |
+|------------------------------|------------------------------------------------------------------|------------------------------------------------------------|---------------------|
+| Bookie Ledgers Count         | meter_bookkeeper_bookie_ledgers_count                            | The number of the bookie ledgers.                          | Bookkeeper Cluster  |
+| Bookie Ledger Writable Dirs  | meter_bookkeeper_bookie_ledger_writable_dirs                     | The number of writable directories in the bookie.          | Bookkeeper Cluster  |
+| Bookie Ledger Dir Usage      | meter_bookkeeper_bookie_ledger_dir_data_bookkeeper_ledgers_usage | The number of successfully created connections.            | Bookkeeper Cluster  |
+| Bookie Entries Count         | meter_bookkeeper_bookie_entries_count                            | The number of failed connections.                          | Bookkeeper Cluster  |
+| Bookie Write Cache Size      | meter_bookkeeper_bookie_write_cache_size                         | The total number of closed connections.                    | Bookkeeper Cluster  |
+| bookie_write_cache_count     | meter_bookkeeper_bookie_write_cache_count                        | The usage of jvm buffer pool.                              | Bookkeeper Cluster  |
```

Comment:
```suggestion
| Bookie Write Cache Count    | meter_bookkeeper_bookie_write_cache_count                        | The usage of jvm buffer pool.                              | Bookkeeper Cluster  |
```

Reply:
I'm a little confused `Bookie Write Cache Size` means `The total number of closed connections. `